_guides/infantry-core.md

```diff
@@ -44,15 +44,15 @@ This handbook is therefore held as a basis of expected in-game performance and b
   * Helps each other with covering different areas, digging trenches, clearing buildings and buddy aid.
 
 
-<h2 id="the-start-of-the-mission">The start of the mission</h2>
+<h2 id=\"the-start-of-the-mission\">The start of the mission</h2>
 
 The start of the mission is crucial. Before stepping out, everybody should know what they will be doing. As such, SLs should take a moment to sit down with his squad and **explain the plan and how he plans to execute it.**
 Radio checks and equipment checks should be made.
 Some missions won’t give this time to the players after the briefing. If this is the case, extra time during the briefing should be allowed so that the SLs can explain to their squads what to do.
 
 ## Spacing
 The default spacing is **10m**. You can use the DUI radar to estimate your distance:
-![alt_text](http://content.globalconflicts.net/guides/infantry-core/dui-spacing.png "image_tooltip")
+![alt_text](http://content.globalconflicts.net/guides/infantry-core/dui-spacing.png \"image_tooltip\")
```

Comment:
I saw you had just fixed the infantry guide in a previous update yesterday. 
By that point I had already put in this request. Is there a way to deny a single commit in the PR, so we can just pull your infantry guide fix?
Or just merge my PR to the branch, and then push your fix again?

Reply:
if you update the branch you are trying to PR it will update the PR request, so just copy paste the file from the current branch and update it in your branch, then this change will disappear (because it will be the same in both branches)